package.json

```diff
@@ -446,7 +447,7 @@
     "ts-dedent": "2.0.0",
     "ts-jest": "29.1.0",
     "ts-morph": "16.0.0",
-    "ts-node": "10.9.2",
+    "ts-node": "10.9.1",
```

Comment:
`ts-node` has been updated on main. Can you resolve this so the package isn't downgraded in this PR.